src/main/java/org/broadinstitute/hellbender/tools/dragstr/CalibrateDragstrModel.java

```diff
@@ -48,7 +51,6 @@
         oneLineSummary = "estimates the parameters for the DRAGstr model",
         programGroup = ShortVariantDiscoveryProgramGroup.class
 )
-@BetaFeature
```

Comment:
If we are going to remove the beta label we should really add more tests to this class. Currently there are no tests for this class. I think we should at least make some integration tests that assert basic outputs for this tool... maybe we can produce an STR list for one of or small reference files?

Reply:
I have an integration test class in a separate branch.... will make a PR of it or perhaps add it here.

Reply:
I have added an Integration test based on the test chr20 chr21 samples that we have in large.